site/about.html

```diff
@@ -1,28 +1,42 @@
+<!doctype html>
+
 <html>
 	<head>
 		<title>About me</title>
-		<link rel="stylesheet" href="style.css" >
+
+		<link rel="stylesheet" href="style.css">
+		<link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
+		<link href='https://fonts.googleapis.com/css?family=Lato:400,900' rel='stylesheet' type='text/css'>
 	</head>
 
 	<body>
-		<nav>
-			<a href="about.html">About Me</a>
-			<a href="portfolio.html">Portfolio</a>
-		</nav>
+		<ul class="top_nav">
```

Comment:
Great use of classes in this block!  For next time, think about nesting the ```<ul>``` tag inside a ```<nav>``` tag.  It seems extraneous on the outside, but it is helpful because it tells the browser that this block is a block of navigation.  Screen readers, for example, will use that information to decide how they will read the links inside this block for their users.